packages/components/src/Bottega.purs

```diff
@@ -50,14 +59,15 @@ getPackages = callApi packagesApi "packageGet" [] {}
 newtype OrderNumber = OrderNumber String
 
 type Order =
-  { number :: OrderNumber
-  , user   :: UUID
-  , status :: OrderStatus
+  { number     :: OrderNumber
+  , user       :: UUID
+  , status     :: OrderStatus
   }
 
 type OrderStatus =
-  { state  :: OrderStatusState
-  , time   :: String
+  { state      :: OrderStatusState
+  , time       :: String
+  , failReason :: Maybe OrderStatusFailReason
```

Comment:
Actually, because we know `failReason` only exists when the state is failed, we could just add it to the `OrderFailed` constructor (instead of having it in the `OrderStatus` record):
```
data OrderStatusState
  = OrderCreated
  | OrderStarted
  | OrderCompleted
  | OrderFailed OrderStatusFailReason
  | OrderCanceled
  | UnknownState
```